scripts/test-installer-ux.sh

```
@@ -0,0 +1,375 @@
+done
+
+mkdir -p "$tmp/no-hermes-bin"
+cp "$tmp/fake-bin/curl" "$tmp/no-hermes-bin/curl"
+node_bin_dir=$(dirname "$(command -v node)")
+env "${common_env[@]}" \
+  PATH="$tmp/no-hermes-bin:$node_bin_dir:/usr/bin:/bin" \
+  HERMES_HOME="$tmp/home/.hermes-without-cli" \
+  INSTALLER_TEST_HERMES_ARCHIVE="$hermes_archive" \
+  bash "$tmp/remote-launcher/install.sh" --non-interactive --with hermes > "$tmp/hermes-without-cli.out"
+grep -q 'Hermes Agent plugin installed' "$tmp/hermes-without-cli.out"
+! grep -q 'Hermes Agent configured' "$tmp/hermes-without-cli.out"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)